docs/index.yaml

```diff
@@ -14,7 +26,7 @@ entries:
     - https://team-blaze.github.io/helm-sqlproxy-ssh-tunnel/sqlproxy-ssh-tunnel-0.2.4.tgz
     version: 0.2.4
   - apiVersion: v1
-    created: "2023-07-04T17:41:49.238912+01:00"
+    created: "2024-10-02T14:58:59.928613511+01:00"
```

Comment:
I assume these changes to the `created` were edited by a script?

Reply:
They were